codec/builtin/build.gradle

```diff
@@ -0,0 +1,14 @@
+plugins {
+    id 'java-library'
+    id 'io.deephaven.project.register'
+}
+
+description 'Codec Builtin: Deephaven builtin codec implementations'
+
+dependencies {
+    implementation project(":codec-base"),
+                   project(":DataStructures"),
```

Comment:
I'm sad that we need dependency on DataStructurces, which has a bunch more transitive dependencies, just so we can depend on `io.deephaven.datastructures.util.CollectionUtil#ZERO_LENGTH_BYTE_ARRAY`. I think we should cleanup these "mostly extraneous" dependencies and simplify it (just create a package private empty byte[] IMO).

Reply:
Colin may be cleaning up `CollectionUtil` for unrelated reasons very soon.